pkg/kube/kube.go

```diff
@@ -226,3 +231,44 @@ func (k *KubeClient) LabelExistsInCluster(ctx context.Context, label string) err
 	}
 	return nil
 }
+
+func (k *KubeClient) CreateSecret(ctx context.Context, name, key, value string, namespace string) error {
+	cmd := []string{
+		"create",
+		"secret",
+		"generic",
+		name,
```

Comment:
Does this mean that only one git token can be stored in the cluster?

Reply:
The logic currently needs only one git token. This `name` variable could be any valid string. We could use it to create as many secrets as we need.